test/e2e/app-dir/use-cache-route-handler-only/app/node/route.ts

```diff
@@ -5,8 +5,6 @@ async function getCachedRandom() {
 
 export async function GET() {
   const rand1 = await getCachedRandom()
-  // TODO: Remove this extra micro task when bug in use cache wrapper is fixed.
-  await Promise.resolve()
```

Comment:
I don't think you can remove this when you rebase because I had to add back the hack while fixing another issue.